manifests/supervisorcluster/1.22/cns-csi.yaml

```diff
@@ -415,7 +415,7 @@ data:
   "block-volume-snapshot": "false"
   "sibling-replica-bound-pvc-check": "true"
   "tkgs-ha": "true"
-  "list-volumes": "false"
+  "list-volumes": "true"
```

Comment:
do you want to enable this feature on the master branch? if not revert it to false.

Reply:
It was only for testing. I have reverted it back to false.